src/Sylius/Bundle/UserBundle/Form/Type/UserType.php

```diff
@@ -38,8 +41,29 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
             ])
             ->add('enabled', CheckboxType::class, [
                 'label' => 'sylius.form.user.enabled',
+                'required' => false,
+            ])
+            ->add('verifiedAt', DateTimeType::class, [
+                'widget' => 'single_text',
+                'label' => 'sylius.form.user.verified_date',
+                'required' => false,
             ])
         ;
+
+        $builder->addEventListener(FormEvents::PRE_SET_DATA, function(FormEvent $event){
+            $data = $event->getData();
+            if(null !== $data) {
+                if($data->isVerified()) {
```

Comment:
```suggestion
            if(null !== $data && $data->isVerified()) {
```

Reply:
It works when editing user, but when creating new one $data is null so calling function isVerified() on it creates error.